.github/workflows/license_check.yml

```
@@ -0,0 +1,39 @@
+        uses: actions/checkout@v4
+      - name: Go changes
+        id: go
+        uses: step-security/changed-files@3dbe17c78367e7d60f00d78ae6781a35be47b4a1 # v45.0.1
+        with:
+          files: |
+            go.mod
+            go.sum
+  go-license-check:
+    name: Go License Check
+    needs: prepare
+    if: needs.prepare.outputs.go == 'true' || github.event_name == 'workflow_dispatch'
```

actionlint (1.7.4)

property "go" is not defined in object type {modified: string}

(expression)